workflow-examples/src/test/java/com/redhat/parodos/examples/tekton/TektonWorkFlow.java

```diff
@@ -0,0 +1,164 @@
+package com.redhat.parodos.examples.tekton;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.List;
+
+import com.redhat.parodos.sdk.api.WorkflowApi;
+import com.redhat.parodos.sdk.api.WorkflowDefinitionApi;
+import com.redhat.parodos.sdk.invoker.ApiClient;
+import com.redhat.parodos.sdk.invoker.Configuration;
+import com.redhat.parodos.sdk.model.ArgumentRequestDTO;
+import com.redhat.parodos.sdk.model.ProjectResponseDTO;
+import com.redhat.parodos.sdk.model.WorkFlowDefinitionResponseDTO;
+import com.redhat.parodos.sdk.model.WorkFlowExecutionResponseDTO;
+import com.redhat.parodos.sdk.model.WorkFlowRequestDTO;
+import com.redhat.parodos.sdk.model.WorkRequestDTO;
+import com.redhat.parodos.workflow.utils.CredUtils;
+import org.junit.Test;
+
+import org.springframework.http.HttpHeaders;
+
+import static com.redhat.parodos.sdkutils.WorkFlowServiceUtils.getProjectAsync;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+public class TektonWorkFlow {
+
+	private final String projectName = "project-1";
+
+	private final String projectDescription = "Tekton example project";
+
+	private final String workflowName = "kubeapiWorkFlow";
+
+	private final String taskName = "kubeapiTask";
+
+	@Test
+	public void runFlow() {
+		ApiClient defaultClient = Configuration.getDefaultApiClient();
+
+		defaultClient.addDefaultHeader(HttpHeaders.AUTHORIZATION, "Basic " + CredUtils.getBase64Creds("test", "test"));
+
+		try {
+			ProjectResponseDTO testProject = getProjectAsync(defaultClient, projectName, projectDescription);
+
+			// GET workflow DEFINITIONS
+			WorkflowDefinitionApi workflowDefinitionApi = new WorkflowDefinitionApi(defaultClient);
+			List<WorkFlowDefinitionResponseDTO> simpleSequentialWorkFlowDefinitions = workflowDefinitionApi
```

Comment:
pls replace with 
```
TestComponents components = new WorkFlowTestBuilder().withDefaultProject().withWorkFlowDefinition(WORKFLOW_NAME)
				.build();
```
which does that already.
See any other workflow in the integration-tests

Reply:
I will try

Reply:
I can not use it. It is from a different module that the examples should not rely on. The examples are equivalent to a customer's project/module. Customer should not depend on our integration tests